src/app/main/connection-window/connection-window.component.ts

```diff
@@ -65,6 +77,8 @@ export class ConnectionWindowComponent extends SubscriptionHandler implements On
   }
 
   onConnect(): void {
+    // When we first connect, we will assume we want to auto connect.
+    localStorage.setItem('auto_login', 'true');
```

Comment:
unit tests should be updated accordingly :)

I can help if needed

Reply:
Yes I would need help. I